frontend/components/dynamic-badge-maker.js

```diff
@@ -62,6 +64,7 @@ function DynamicBadgeMaker({ baseUrl = document.location.href }) {
       </select>{' '}
       {inputs.map(({ name, placeholder = name }) => (
         <InlineInput
+          key={name}
```

Comment:
Just a question out of curiosity, why is the `key` needed now?

Reply:
Yea, good question. I believe it was a regression in the previous PR: #3096